### PR TITLE
Added missing default values and filter archived

### PR DESCRIPTION
**Pull request by ansvu**

### CODE REVIEW

File: roles/create_helmchart/tasks/get_all_helm_chart_projects_basedon_product_list_id.yml [hunk 1]
```diff
@@ -23,7 +23,7 @@
   set_fact:
     all_helmchart_projects: "{{ cert_project_list_output.json.data | json_query(query) + [new_helmchart_projects] | flatten }}"
   vars:
-    query: "[?product_listings[0] == '{{ cert_listings.pyxis_product_list_identifier }}'] | []._id"
+    query: "[?product_listings[0] == '{{ cert_listings.pyxis_product_list_identifier }}' && project_status != 'archived'] | []._id"
```
Comment (manurodriguez) [inline roles/create_helmchart/tasks/get_all_helm_chart_projects_basedon_product_list_id.yml:26]: I just have concerns about where this `project_status` var is defined, I see it in the `roles/create_certification_project/templates/create_project_*.json.j2` files, but I'm not sure if it comes from there, if you have more insights please let me know.

File: roles/openshift_cnf/tasks/get_all_openshift_cnf_projects_basedon_product_list_id.yml [hunk 2]
```diff
@@ -23,7 +23,7 @@
   set_fact:
     all_openshift_cnf_projects: "{{ cert_project_list_output.json.data | json_query(query) + [new_openshift_cnf_projects] | flatten }}"
   vars:
-    query: "[?product_listings[0] == '{{ cert_listings.pyxis_product_list_identifier }}'] | []._id"
+    query: "[?product_listings[0] == '{{ cert_listings.pyxis_product_list_identifier }}' && project_status != 'archived'] | []._id"
```
Comment (manurodriguez) [inline roles/openshift_cnf/tasks/get_all_openshift_cnf_projects_basedon_product_list_id.yml:26]: same as in the previous comment

**Review by manurodriguez — COMMENTED** [review]
I general, looks safe to me to be merged without testing, I just have a question below. Thanks

File: roles/create_helmchart/tasks/get_all_helm_chart_projects_basedon_product_list_id.yml (hunk 1 above)
Comment (ansvu) [reply roles/create_helmchart/tasks/get_all_helm_chart_projects_basedon_product_list_id.yml:26]: Thanks for your comments and concern, Manny. 
I created simple-play to test locally without dci to filter the archived project:

**Archived project info**
This project is archived and attached to 9GC UDM Product-listing ID: 642a2aff90de6580a53c2f66
ubi8-nginx-custom-name Archived | Container Image | 9GC UDM
archived-project-id: 6514b63aed479d0b32a265f8

**Active project info**
This project is not archived one and using same product-listing ID.
active-project-id: 64cd38e9923cb9faf0c3b2b9

**Now run a simply-play ansible with this original query:**
`query: "[?product_listings[0] == '{{ product_list_id }}'  && project_status != 'archived'] | []._id"
`
**Results**:
expecting only active-project-id on the output:
ok: [localhost] => {
    "certification_ids": [
        "64cd38e9923cb9faf0c3b2b9"
    ]
}

Next test is to add `s` to `projects_status` to see if archived-project-id also output (**expected**):
**Results**:
ok: [localhost] => {
    "certification_ids": [
        "64cd38e9923cb9faf0c3b2b9",
        "6514b63aed479d0b32a265f8" -->archived-project-id
    ]
}
It works like this on other projects. Please let me know. Thanks

Comment (manurodriguez) [reply roles/create_helmchart/tasks/get_all_helm_chart_projects_basedon_product_list_id.yml:26]: Thanks for the explanation Andrew

**Review by manurodriguez — APPROVED** [review]

**Pull request merged**